rmgpy/rmg/main.py

```diff
@@ -898,6 +902,27 @@ def execute(self, **kwargs):
                 
                 plot_sensitivity(self.outputDirectory, index, reactionSystem.sensitiveSpecies)
 
+        # Check all core reactions (in both directions) for collision limit violation
+        logging.info("Checking core reactions for collision rate limit violators...")
+        violators = []
+        for rxn in self.reactionModel.core.reactions:
+            f_violator, r_violator = rxn.check_collision_limit_violation(t_min=self.Tmin, t_max=self.Tmax,
+                                                            p_min=self.Pmin, p_max=self.Pmax)
+            if f_violator is not None:
+                violators.append(f_violator)
+            if r_violator is not None:
+                violators.append(r_violator)
+        if len(violators):
```

Comment:
This can be replaced with `if violators:` since empty lists have evaluate to False. This is not a big deal though